ompi/mpi/c/profile/Makefile.am

```diff
@@ -0,0 +1,484 @@
+# -*- makefile.am -*-
```

Comment:
It looks like this PR re-added the `ompi/mpi/c/profile/Makefile.am` file.  This file has been removed from master -- it may have gotten re-added in this PR as a result of a rebase.  Please check current master HEAD to see how we simplified this a bit.